stackrox/acs-fleet-manager
# ROX-24986: Add Central VerticalPodAutoscaler

## Description
- Added a VerticalPodAutoscaler for Central
- Modified the dev/ci scripts to install the VerticalPodAutoscaler controller

## Checklist (Definition of Done)


## Test manual

- Deploy fleetmanager and fleetshard sync
- Create a new Central
- Observe that the namespace does not contain a VerticalPodAutoscaler for Central
- Edit the gitops config, set `.tenantResources.default.verticalPodAutoscalers.central.enabled` to `true`
- Observe that the namespace contains a `VerticalPodAutoscaler`
- Edit the gitops config, set `.tenantResources.default.verticalPodAutoscalers.central.enabled` to `false`
- Observe that the `VerticalPodAutoscaler` is removed
